src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -67,6 +72,24 @@ private void postCreateCaseInCcdProcess(SyaCaseWrapper appeal, String firstHalfO
             Map<String, byte[]> additionalEvidence = downloadEvidence(appeal);
             roboticsService.sendCaseToRobotics(caseData, caseDetails.getId(), firstHalfOfPostcode, pdf,
                     additionalEvidence);
+
+            attachRoboticsJsonToCaseInCcd(caseData, idamTokens, caseDetails);
+        }
+    }
+
+    private void attachRoboticsJsonToCaseInCcd( SscsCaseData caseData,
+                                                IdamTokens idamTokens, SscsCaseDetails caseDetails) {
+        byte[] pdf = sscsPdfService.generateAndSendPdf(caseData, caseDetails.getId(), idamTokens);
```

Comment:
Why are we generating and sending the PDF again?

Reply:
That's a mistake that I made while resolving the merge conflicts. This got picked up by the unit tests.